src/sentry/utils/sentry_apps/webhooks.py

```diff
@@ -36,6 +39,42 @@ def wrapper(sentry_app, app_platform_event, url=None):
     return wrapper
 
 
+def check_broken(sentryapp: SentryApp, org_id: str):
+    redis_key = sentryapp._get_redis_key(org_id)
+    buffer = IntegrationRequestBuffer(redis_key)
+    if buffer.is_integration_broken():
+        org = Organization.objects.get(id=org_id)
+        if features.has("organizations:disable-sentryapps-on-broken", org):
```

Comment:
Do we know how often webhook requests are being made per second? This feature check could potentially cause some issues with flagr if the throughput is high.

Reply:
There's a dashboard [here](https://app.datadoghq.com/dashboard/dei-yif-h45/integrations-health?fullscreen_end_ts=1691084459921&fullscreen_paused=true&fullscreen_section=overview&fullscreen_start_ts=1691084399921&fullscreen_widget=3336705431871648&from_ts=1688490217830&to_ts=1691082217830&live=true) for internal sentry app requests

Reply:
Double checked on this and we're good - we should only really be worried about event ingestion kinda numbers for that.